### PR TITLE
Updated PyPI replication for v0.10.1.0

### PR DESCRIPTION
**Pull request by lintool**
Hey @MXueguang - I fixed a lot of issues with previous iteration... can you please replication at both HEAD and also `pyserini==0.10.1.0`?

Thanks!

### CODE REVIEW

File: docs/pypi-replication.md [hunk 1]
```diff
@@ -24,81 +25,126 @@ P_30                  	all	0.3102
 
 ## MS MARCO Passage Ranking
 
+Before we can evaluate the output, we need to first fetch the eval script and relevance judgments:
+
+```bash
+$ wget https://raw.githubusercontent.com/castorini/anserini-tools/master/scripts/msmarco/msmarco_eval.py
```
Comment (MXueguang) [inline docs/pypi-replication.md:31]: This url gives `404 Not Found`

File: docs/pypi-replication.md [hunk 2]
```diff
@@ -24,81 +25,126 @@ P_30                  	all	0.3102
...
+#####################
 ```
 
-
 ## MS MARCO Document Ranking
 
+Before we can evaluate the output, we need to first fetch the eval script and relevance judgments:
+
+```bash
+$ wget https://raw.githubusercontent.com/castorini/anserini-tools/master/scripts/msmarco/ms_marco_doc_eval.py
```
Comment (MXueguang) [inline docs/pypi-replication.md:72]: This url gives `404 Not Found` too

File: docs/pypi-replication.md [hunk 3]
```diff
@@ -24,81 +25,126 @@ P_30                  	all	0.3102
...
+$ python ms_marco_doc_eval.py --judgments qrels.msmarco-doc.dev.txt --run run.msmarco-doc.passage-expanded.txt
+#####################
+MRR @100: 0.32081861579183807
+QueriesRanked: 5193
+#####################
+```
+
+## Notes
+
+Prior to v0.10.1.0, the above commands get different results:
```
Comment (MXueguang) [inline docs/pypi-replication.md:142]: we can add a link here point to the v0.10.0.1 replication doc to show the results before?

**Review by MXueguang — CHANGES_REQUESTED** [review]
the url seems not work?

File: docs/pypi-replication.md (hunk 1 above)
Comment (lintool) [reply docs/pypi-replication.md:31]: do'h, I just changed this: https://github.com/castorini/anserini-tools/pull/23

Will fix.

File: docs/pypi-replication.md (hunk 2 above)
Comment (lintool) [reply docs/pypi-replication.md:72]: Same as above.

File: docs/pypi-replication.md (hunk 3 above)
Comment (lintool) [reply docs/pypi-replication.md:142]: Yes, good call. Will do.

**Comment by lintool** [comment]
@MXueguang Ready for re-review.

**Review by MXueguang — APPROVED** [review]
LGTM
replicated on current HEAD and pyserini==0.10.1.0
get exactly same results

**Pull request merged**